megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -4287,10 +4289,20 @@ else if ((atype != null)
         // Unit-specific modifiers
         
         // -1 to hit a SuperHeavy mech
-        if (te != null && (te instanceof Mech) && ((Mech) te).isSuperHeavy()) {
+        if ((te != null) && (te instanceof Mech) && ((Mech) te).isSuperHeavy()) {
             toHit.addModifier(-1, Messages.getString("WeaponAttackAction.TeSuperheavyMech"));
         }
         
+        // large support tanks get a -1 per TW
+        if (te instanceof LargeSupportTank) {
+            toHit.addModifier(-1, Messages.getString("WeaponAttackAction.TeLargeSupportTank"));
+        }
```

Comment:
What about large Fixed-Wing Craft? They don't use LargeSupportTank. I'm also not sure about Support VTOLs being handled by this either